gammapy/datasets/tests/test_make.py

```diff
@@ -72,9 +76,41 @@ def test_make_test_bg_cube_model():
     e_points = bg_cube_model.energy_bin_centers
     x_points, y_points, e_points = np.meshgrid(x_points, y_points, e_points,
                                                indexing='ij')
-    det_bin_index = bg_cube_model.find_det_bin(Angle([x_points, y_points]))
+    det_bin_index = bg_cube_model.find_coord_bin(Angle([x_points, y_points]))
     e_bin_index = bg_cube_model.find_energy_bin(e_points)
-    bg = bg_cube_model.background[e_bin_index, det_bin_index[1], det_bin_index[0]]
+    bg = bg_cube_model.data[e_bin_index, det_bin_index[1], det_bin_index[0]]
 
     # assert that values are 0
     assert_quantity_allclose(bg, Quantity(0., bg.unit))
+
+
+def test_make_test_dataset(tmpdir):
+    # create a dataset
+    fits_path = str(tmpdir.join('test_dataset'))
```

Comment:
The idea is to put inputs and outputs in separate folders in order not to mix up things. So I'm not changing this.